### ISSUE TITLE
The `change` event is sent twice on change

### ISSUE THREAD

**Issue by AlexandreBonneau** [autoNumeric/autoNumeric#550]
### Expected behavior
When modifying the element value, then blurring it, a single `change` event should be caught.

### Actual behavior
Two such events are sent

### Steps to reproduce the problem
This is reproducible with `v4.1.0-beta.25`, on [this codepen](https://codepen.io/AnotherLinuxUser/pen/BweJOv?editors=1011).

**Comment by ducman** [comment]
I've looked into this a bit and the change event is firing at [line 6519](https://github.com/autoNumeric/autoNumeric/blob/master/src/AutoNumeric.js#L6519) and also inside [line 6524](https://github.com/autoNumeric/autoNumeric/blob/master/src/AutoNumeric.js#L6524) which is a couple of lines down.

I'm can only guess that on blur should be firing a blur event instead of a change event? 

Hope that helps.

**Comment by AlexandreBonneau** [comment]
Indeed, something is fishy here...
:innocent:

**Comment by AlexandreBonneau** [comment]
Note: as a related bug, if you have an AutoNumeric element initialized with the following options:
```js
new AutoNumeric('.an', {
    negativeBracketsTypeOnBlur: AutoNumeric.options.negativeBracketsTypeOnBlur.parentheses,
});
```
and if its value is negative (ie. with parenthesis when unfocused), then when focusing then blurring the input without changing its value, **will** output 2 changes events.

This is wrong and will be corrected as well.

**Comment by AlexandreBonneau** [comment]
Fixed in `v4.2.5`
